org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/tree/ITree.java

```diff
@@ -406,6 +406,8 @@ default void addUITreeListener(TreeListener listener, Integer... eventTypes) {
 
   void setNodesChecked(List<ITreeNode> nodes, boolean checked, boolean enabledNodesOnly, boolean forceUpdateNode);
 
+  void setAllNodesChecked(boolean checked);
```

Comment:
Maybe also add checkAllNodes() and uncheckAllNodes() which call setAllNodesChecked